subcommittee/coding-guidelines/initiatives/safe-use-of-unsafe-guidelines/rust-embedded-lld-safe-definition.md

```diff
@@ -0,0 +1,295 @@
+# Rust Embedded Low Level Drivers and Safe Definition
```

Comment:
Would be good to put images under `rust-embedded-lld-safe-definitions-images` so that if we have other documents with images they can go under a corresponding folder.

Reply:
Done